src/components/project/show.vue

```diff
@@ -25,6 +25,7 @@ except according to the terms contained in the LICENSE file.
           role="presentation">
           <router-link :to="tabPath('datasets')">
             {{ $t('resource.datasets') }}
+            <span class="chip">New</span>
```

Comment:
"New" needs an i18n message I think. I think it could go in en.json5 with a comment mentioning that it will be shown next to new features.

Also, based on the screenshot in getodk/central#450, I think this should be lowercase.

Reply:
Aly requested to capitalize it